handlers/namespaces.go

```diff
@@ -55,3 +57,34 @@ func getNamespaceMetrics(w http.ResponseWriter, r *http.Request, promSupplier pr
 	metrics := prom.GetMetrics(&params)
 	RespondWithJSON(w, http.StatusOK, metrics)
 }
+
+//NamespaceValidations the API handler to fetch validations to be displayed.
```

Comment:
Two things:
- Typo, missing space at the beginning.
- This endpoint won't return validations, there is an existing endpoint that returns istio objects + validations. This enpoint returns a summary.

I can understand we don't want the existing endpoint as we only need the summary, but description should clarify that to avoid misunderstandings.